### PR TITLE
[4.0] Namespace mod articles categories

### PR DESCRIPTION
**Pull request by C-Lodder**
### Summary of Changes

Namespace mod articles categories

### CODE REVIEW

File: modules/mod_articles_categories/mod_articles_categories.php [hunk 1]
```diff
@@ -9,28 +9,28 @@
...
 
-JLoader::register('JCategoryNode', JPATH_BASE . '/libraries/legacy/categories/categories.php');
-
 $cacheid = md5($module->id);
 
-$cacheparams               = new stdClass;
+$cacheparams               = new \stdClass;
 $cacheparams->cachemode    = 'id';
-$cacheparams->class        = 'ModArticlesCategoriesHelper';
+$cacheparams->class        = 'Joomla\Module\ArticlesCategories\Site\Helper\ArticlesCategoriesHelper';
```
Comment (laoneo) [inline modules/mod_articles_categories/mod_articles_categories.php:22]: You can also do here ArticlesCategoriesHelper::class

Comment (C-Lodder) [reply modules/mod_articles_categories/mod_articles_categories.php:22]: So `$cacheparams->class = 'ArticlesCategoriesHelper::class';` ?

Comment (laoneo) [reply modules/mod_articles_categories/mod_articles_categories.php:22]: Without '

Comment (laoneo) [reply modules/mod_articles_categories/mod_articles_categories.php:22]: It's just cosmetic, leave it as it is, np

**Pull request merged**